module/canvas/token-placement.mjs

```diff
@@ -10,6 +10,9 @@
  *
  * @typedef {object} PlacementData
  * @property {PrototypeToken} prototypeToken
+ * @property {object} index
+ * @property {number} index.total             Index of the placement across all placements.
```

Comment:
It seems we don't actually use this total?

Reply:
No, but I figured since it is easy to make it might be nice for other places to have this bit of data in hooks, in places where they don't have the whole `placements` array to work with.

Reply:
Sure, that's no problem then